### ISSUE TITLE
OTP support in the extension

### ISSUE THREAD

**Issue by jaydisc** [mssun/passforios#180]
Unfortunately, there doesn't seem to be a way to get an OTP from the extension?

**Comment by yishilin14** [comment]
Good point. I think when both password and opt are detected in an entry, Pass should ask which password should be copied.
